client/src/components/TodayScheduleTable/index.js

```diff
@@ -0,0 +1,367 @@
+        newData.splice(index, 1);
+        // setSrvData(newData);
+        setEditingKey('');
+      }
+    } catch (errInfo) {
+      setError(
+        errInfo.response
+          ? errInfo.response.data.message
+          : `Something went wrong`
+      );
+    }
+  };
+
+  const columns = [
+    {
+      title: 'Stats',
+      dataIndex: 'isDone',
+      defaultSortOrder: 'ascend',
+      width: '5%',
+      sorter: {
+        compare: (a, b) => a.isDone - b.isDone,
+        multiple: 3,
+      },
+      render: (_, { key, isDone }) => (
+        <Popconfirm
+          disabled={isDone}
+          title="Sure to cancel?"
+          onConfirm={() => {
+            check(key);
+          }}
+          onCancel={() => {
+            setChecked(false);
+          }}
+        >
+          <Checkbox
+            disabled={isDone}
+            checked={isDone && checked}
+            onChange={() => {
+              setChecked(true);
+            }}
+          />
+        </Popconfirm>
+      ),
+    },
+    {
+      title: 'first Name',
+      dataIndex: 'firstName',
+      width: '15%',
+    },
+    {
+      title: 'last Name',
+      dataIndex: 'lastName',
+      width: '15%',
+    },
+    {
+      title: 'age',
+      dataIndex: 'age',
+      width: '5%',
+    },
+    {
+      title: 'Appointment Date',
+      dataIndex: 'appointmentDate',
+      width: '17%',
+      sorter: {
+        compare: (a, b) => a.age - b.age,
+        multiple: 3,
+      },
+      render: (text, { key, appointmentDate }) => {
+        if (editingKey && editingKey === key) {
+          return (
+            <DatePicker
+              defaultValue={moment(appointmentDate, 'YYYY-MM-DD')}
+              format="YYYY-MM-DD"
+              onChange={onDateChange}
+            />
+          );
+        }
+        return text;
+      },
+    },
+    {
+      title: 'Appointment time',
+      dataIndex: 'appointmentTime',
+      width: '17%',
+      defaultSortOrder: 'ascend',
+      sorter: {
+        compare: (a, b) =>
+          moment(a.appointmentTime, 'hh:mm:ss') -
+          moment(b.appointmentTime, 'hh:mm:ss'),
+        multiple: 3,
+      },
+      render: (text, { key, appointmentTime }) => {
+        if (editingKey && editingKey === key) {
+          return (
+            <TimePicker
+              defaultValue={moment(appointmentTime, 'hh:mm:ss')}
+              onChange={onTimeChange}
+            />
+          );
+        }
+        return text;
+      },
+    },
+    {
+      title: 'operation',
+      dataIndex: 'operation',
+      render: (_, record) => {
+        const editable = isEditing(record);
+        return (
+          <div>
+            <Popconfirm
+              title="Sure to Delete?"
+              disabled={record.isDone}
+              onConfirm={() => deleteCell(record.key)}
+            >
+              <DeleteOutlined
+                style={{ color: 'red', fontSize: '21px', margin: '0 10px' }}
```

Comment:
Please to move the styles into css file - please don't make inline styling.

Reply:
done